backends.yaml

```
@@ -1,4 +0,0 @@
-backends:
-- name: mapreduce-backend
-  instances: 10
```

```suggestion
  instances: 10```suggestion
  instances: 10```suggestion
  instances: 10```suggestion
  instances: 10```suggestion
  instances: 10********_____[[```suggestion
  instances: 10
```
]()]()_____********
```

```

```

````suggestion
  instances: 10```suggestion
  instances: 10```suggestion
  instances: 10```suggestion
  instances: 10```suggestion
  instances: 10```suggestion
  instances: 10
```

```

```

```

```

```
``

```